lib/rex/post/meterpreter/ui/console/command_dispatcher/core.rb

```diff
@@ -322,15 +326,43 @@ def cmd_interact(*args)
 
   alias cmd_interact_tabs cmd_close_tabs
 
+  def cmd_irb_help
+    print_line "Usage: irb"
+    print_line
+    print_line "Execute commands in a Ruby environment"
+    print @@irb_opts.usage()
+  end
+
   #
   # Runs the IRB scripting shell
   #
   def cmd_irb(*args)
-    print_status("Starting IRB shell")
-    print_status("The 'client' variable holds the meterpreter client\n")
+    expressions = []
+
+    # Parse the command options
+    @@irb_opts.parse(args) do |opt, idx, val|
+      case opt
+        when '-e'
```

Comment:
actually, I can kill 2 birds here by inverting the logic and including the regular irb stuff in that if statement